docs/rules/no-unsafe-finally.md

```diff
@@ -0,0 +1,134 @@
+# disallow control flow statements in `finally` blocks (no-unsafe-finally)
+
+Javascript suspends the control flow statements of `try` and `catch` blocks until the execution of `finally` block finishes. So, when a control flow statement is used in a `finally` block, it overwrites the previous statements, which is considered as unexpected behavior. Examples of this situation:
+
+When `return` is used:
```

Comment:
Seems like all four control statements behave the same in finally. So instead of writing them separately, may I just say :

"When `return`, `throw`, `break` or `continue` is used in `finally`, control flow statements inside `try` and `catch` are overwritten, such as: ",
then give several examples with their explanations?